src/applications/healthcare/questionnaire/config/form.js

```diff
@@ -25,6 +28,18 @@ const formConfig = {
     });
   },
   formId: VA_FORM_IDS.FORM_HC_QSTNR,
+  saveInProgress: {
+    messages: {
+      inProgress: 'Your upcoming appointment questionnaire is in progress.',
+      // TODO: Fix the expired message
+      expired:
+        'Your saved upcoming appointment questionnaire has expired. If you want to apply for appointment questionnaire, please start a new application.',
+      saved: 'Your appointment questionnaire application has been saved.',
+    },
+  },
+  customText: {
+    appType: 'questionnaire',
+  },
```

Comment:
Do we need to specify the `appType` if we are also providing messages that override the default messages?

Reply:
Yeah; the `appType` is used for more than these messages. I have to verify with the owners of this form that this is an okay addition. It's totally not necessary for this PR (though it was in a previous iteration).